T5-G2/t5/src/main/resources/templates/main.html

```diff
@@ -1,79 +1,187 @@
-                            </div>
-                            <!--Modifica (20/05): Aggiunta nuova voce all'elenco delle modalità di gioco-->
-                            <div class="col-md-4">
-                                <button type="button" class ="lista-item my-button" onclick="selectGameMode('Scalata')"> <i class="fa-solid fa-stairs" style="color: #f5b041;"></i>   Scalata</button>
-                            </div>
-                            <div class="col-md-4">
-                                <button type="button" class ="lista-item my-button" onclick="selectGameMode('LeaderboardScalata')"> <i class="fa-solid fa-trophy" style="color: #f5b041;"></i>   LeaderboardScalata</button>
-                            </div>
-                            <!--Fine modifica (20/05)-->
-                        </span> 
-                        </div>
-                        <div class="name"></div>
+    <div class="modal fade" id="Scelta" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h1 class="modal-title fs-5" id="exampleModalLabel">Seleziona Opzioni</h1>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                </div>
+                <div class="modal-body">
+                    <!-- Select per scegliere la classe -->
+                    <div class="mb-3">
+                        <label for="selectClasse" class="form-label">Seleziona Classe</label>
+                        <select class="form-select" id="selectClasse" aria-label="Seleziona Classe">
+                            <option selected>Calcolatrice</option>
+                            <option value="1">Classe 1</option>
+                            <option value="2">Classe 2</option>
+                        </select>
+                    </div>
+
+                    <!-- Select per scegliere la difficoltà -->
+                    <div class="mb-3">
+                        <label for="selectDifficolta" class="form-label">Seleziona Difficoltà</label>
+                        <select class="form-select" id="selectDifficolta" aria-label="Seleziona Difficoltà">
+                            <option selected>Facile</option>
+                            <option value="1">Intermedio</option>
+                            <option value="2">Difficile</option>
+                        </select>
+                    </div>
 
+                    <!-- Select per scegliere il tipo di robot -->
+                    <div class="mb-3">
+                        <label for="selectRobot" class="form-label">Seleziona Tipo di Robot</label>
+                        <select class="form-select" id="selectRobot" aria-label="Seleziona Tipo di Robot">
+                            <option selected>Robot Randloop</option>
```

Comment:
Robot Randoop

Reply:
Nella Main page abbiamo messo delle schede ogni una è una modalità di gioco, questo è il modal della prima modalità, per ora i dati sono statici ma integreremo i componenti per prendere in automatico le ClassUT e così via.

Reply:
Si, stavo vedendo, mi riferivo semplicemente ad un refuso nel nome del robot.